### PR TITLE
[box] Xenobridge always starts connected

### PR DESCRIPTION
**Pull request by patpol4**
# General Documentation

### Intent of your Pull Request
Removes the space variation from xenobridge, now there will always be a passage
image

### Why is this change good for the game?
No uncertainty if you're going to be able to cross from one maint to another because of random chance.
Also I don't like it

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Removes the xenobridge space variation

### What should players be aware of when it comes to the changes your PR is implementing?
the xenobridge space variation is removed
### What general grouping does this PR fall under? 
mapping

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
not really
### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
nope

# Changelog

:cl:  
rscdel: random variations gone from xenobridge
tweak: xenobridge is always connected
/:cl:

### CODE REVIEW

**Comment by enkidienne** [comment]
its just kinda annoying not to have it there

**Review by JamieD1 — APPROVED** [review]

**Comment by patpol4** [comment]
lol
